Vic2ToHoI4/Data_Files/blankMod/output/common/on_actions/00_on_actions.txt

```diff
@@ -371,18 +343,19 @@ on_actions = {
 	}
 	
 	on_coup_succeded = {
+	###Democratic may not always exist. Nevertheless, civil-war created democratic nations should gain elections
 		effect = {
-			#Turn elections on for democracies created from coup
-			random_other_country = {
-				limit = {
-					has_government = democratic
-					original_tag = ROOT
-				}
-				set_politics = {
-					ruling_party = democratic
-					elections_allowed = yes
-				}
-			}
+		#	#Turn elections on for democracies created from coup
```

Comment:
Almost done, but delete instead of commenting out (feel free to leave a comment explaining why).

Reply:
Done